src/style.css

```diff
@@ -1 +1,75 @@
-/* Styles go here */
+html {
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+}
+
+body {
+  margin: 0;
+}
+
+.header {
+  width: 100%;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.logo {
+  width: 40px;
+  height: 40px;
+}
+
+.image-link {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  margin-left: 50px;
+}
+
+.nav {
+  margin-right: 40px;
+}
+
+.menu {
+  display: flex;
+  list-style: none;
+  padding: 0;
+  justify-content: center;
+  margin: 0;
+}
+
+.menu-option {
+  display: flex;
+  align-items: center;
+  text-align: center;
+  margin: 0 10px;
```

Comment:
Again, this thing break tests, set margins to one side only and disable them for last element
If your container wasn't flexbox, you also could face margin collapse